subsys/logging/log_core.c

```diff
@@ -488,6 +486,11 @@ u32_t log_buffered_cnt(void)
 	return buffered_cnt;
 }
 
+void log_dropped(void)
```

Comment:
Why this is not static function in in `log_msg.c` ?

Reply:
it increments variable that is static in log_core and is used by log_core internally.